Example/main.cpp

```diff
@@ -50,8 +50,8 @@ int main(int argc, char **argv) {
 
         // use the data
         std::vector<float> data;
-        uint channel = 1;
-        uint n_samples = 10;
+        unsigned intchannel = 1;
+        unsigned intn_samples = 10;
```

Comment:
Error when running make:
```
❯ make
Scanning dependencies of target ZMQReader
[ 20%] Building CXX object CMakeFiles/ZMQReader.dir/ZMQReader.cpp.o
[ 40%] Building CXX object CMakeFiles/ZMQReader.dir/ZMQRegister.cpp.o
[ 60%] Linking CXX static library libZMQReader.a
[ 60%] Built target ZMQReader
Scanning dependencies of target OpenEphysZMQReader
[ 80%] Building CXX object CMakeFiles/OpenEphysZMQReader.dir/main.cpp.o
/home/anjal/OpenEphys/OEPlugins/ZMQInterface/Example/main.cpp: In function ‘int main(int, char**)’:
/home/anjal/OpenEphys/OEPlugins/ZMQInterface/Example/main.cpp:55:36: error: ‘channel’ was not declared in this scope
         if(reader.get_channel_data(channel, n_samples,  data)){
                                    ^~~~~~~
/home/anjal/OpenEphys/OEPlugins/ZMQInterface/Example/main.cpp:55:36: note: suggested alternative: ‘intchannel’
         if(reader.get_channel_data(channel, n_samples,  data)){
                                    ^~~~~~~
                                    intchannel
/home/anjal/OpenEphys/OEPlugins/ZMQInterface/Example/main.cpp:55:45: error: ‘n_samples’ was not declared in this scope
         if(reader.get_channel_data(channel, n_samples,  data)){
                                             ^~~~~~~~~
/home/anjal/OpenEphys/OEPlugins/ZMQInterface/Example/main.cpp:55:45: note: suggested alternative: ‘intn_samples’
         if(reader.get_channel_data(channel, n_samples,  data)){
                                             ^~~~~~~~~
                                             intn_samples
make[2]: *** [CMakeFiles/OpenEphysZMQReader.dir/build.make:82: CMakeFiles/OpenEphysZMQReader.dir/main.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:136: CMakeFiles/OpenEphysZMQReader.dir/all] Error 2
make: *** [Makefile:149: all] Error 2
```
**Fix**: Needs space between `int` and variable name.